package.json

```diff
@@ -91,5 +94,8 @@
         "compile": "tsc -p ./",
         "update-vscode": "node ./node_modules/vscode/bin/install",
         "postinstall": "node ./node_modules/vscode/bin/install"
+    },
+    "lint-staged": {
+        "**/*": "prettier --write --ignore-unknown"
```

Comment:
How about `prettier --write --ignore-unknown src/*.js` instead of having a large `.prettierignore` file?